### PR TITLE
OLS-1284: OpenShift Lightspeed Release Notes 0.2.1

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Affects:
[lightspeed-main](https://github.com/openshift/openshift-docs/tree/lightspeed-docs-main)
[lightspeed-docs-1.0tp1](https://github.com/openshift/openshift-docs/tree/lightspeed-docs-1.0tp1)

This PR is part of the standalone doc set for the Lightspeed project. Kathryn is aware that content applies for a product that is part of a Tech Preview release. The project is seeking feedback from early adopters. 

PR must be CP'd back to the TP branch.

Version(s): TP

Issue: https://issues.redhat.com/browse/OLS-1284

Link to docs preview:
https://85894--ocpdocs-pr.netlify.app/openshift-lightspeed/latest/release_notes/ols-release-notes.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/ols-0-2-1-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+{ols-official} 0.2.1 is now available on {ocp-product-title} 4.15 or later.
+
+[id="ols-0-2-1-enhancements_{context}"]
+== Enhancements
+
+The following enhancements have been made with {ols-official} 0.2.1:
+
+* With this release, the {ols-long} User Interface is hidden if the *Hide Lightspeed* option is selected in *User Preferences*.
+** Xavier to review in latest product build
```
Comment (xrajesh) [inline modules/ols-0-2-1-release-notes.adoc:16]: @kyoto I am not seeing this option in *User Preferences*. when I installed v0.2.1 Could you please check

File: modules/ols-0-2-1-release-notes.adoc [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+* This release adds the ability to attach a YAML file to a question you ask on any page accessible from *Virtualization* in the {ocp-product-title} web console. This release also adds the ability to attach events and logs for `VirtualMachines` and `VirtualMachineInstances`.
+
+* Beginning in this release you can set the `api_version` parameter for the {azure-openai} provider in the `OLSConfig` Custom Resource.
+** OLS-1092
+
+* This release provides the ability to make use of the cluster-wide Transport Layer Security (TLS) profile and to make use of the TLS custom profile for the {ols-long} stack.
+** OLS-1097
+
+* This release also adds the ability to use user provided TLS certificates for encrypting communication between {ols-long} services.
```
Comment (xrajesh) [inline modules/ols-0-2-1-release-notes.adoc:28]: This release adds ability for the user to provide their own TLS certificates for the communication between console proxy and the application HTTPS endpoint.

File: modules/ols-0-2-1-release-notes.adoc [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+** OLS-385
+
+* With this release, the {ols-long} User Interface displays an email link that can be used to contact the TLS profile and {ols-official} Development Team.
```
Comment (xrajesh) [inline modules/ols-0-2-1-release-notes.adoc:31]: * With this release, the {ols-long} User Interface displays an email link that can be used to contact the  {ols-official} Development Team.

Comment (xrajesh) [reply modules/ols-0-2-1-release-notes.adoc:31]: @rh-tokeefe Please remove the story numbers for the text too

File: modules/ols-0-2-1-release-notes.adoc [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+* The text area of the {ols-long} User Interface now generates a preview when you attach an event to a prompt. The preview presents the actual YAML text for the event, helping you determine if attaching the event will provide additional context for the question being asked.
+
+* This release adds the ability to attach a YAML file to a question you ask on any page accessible from *Virtualization* in the {ocp-product-title} web console. This release also adds the ability to attach events and logs for `VirtualMachines` and `VirtualMachineInstances`.
+
+* Beginning in this release you can set the `api_version` parameter for the {azure-openai} provider in the `OLSConfig` Custom Resource.
+
+* This release provides the ability to make use of the cluster-wide Transport Layer Security (TLS) profile and to make use of the TLS custom profile for the {ols-long} stack.
+
+* This release adds ability for users to provide their own TLS certificates for communication between the {ols-long} console and the {ols-long} service.
```
Comment (aspauldi) [inline modules/ols-0-2-1-release-notes.adoc:23]:
```suggestion
* This release adds the ability for users to provide their own TLS certificates for communication between the {ols-long} console and the {ols-long} service.
```

**Comment by aspauldi** [comment]
One comment about adding an article; other than that, LGTM!

**Comment by stevsmit** [comment]
lgtm

**Pull request merged**